go/libkb/config.go

```diff
@@ -605,7 +599,28 @@ func (f *JSONConfigFile) SetUpdatePreferenceSkip(v string) error {
 }
 
 func (f *JSONConfigFile) SetUpdatePreferenceSnoozeUntil(t keybase1.Time) error {
-	path := "updates.snooze"
+	return f.SetTime("updates.snooze", t)
+}
+
+func (f *JSONConfigFile) SetUpdateLastChecked(t keybase1.Time) error {
+	return f.SetTime("updates.last_checked", t)
+}
+
+func (f JSONConfigFile) GetTime(path string) keybase1.Time {
```

Comment:
Can you call this `GetTimeAtPath` to be consistent with the other functions here?